Makefile

```diff
@@ -143,16 +143,17 @@ TF_CFLAGS_aarch64	=	-mgeneral-regs-only -mstrict-align
 ASFLAGS_aarch32		=	-march=armv8-a
 TF_CFLAGS_aarch32	=	-march=armv8-a
 
-ASFLAGS			+= 	-nostdinc -ffreestanding -Wa,--fatal-warnings	\
-				-Werror -Wmissing-include-dirs			\
-				-D__ASSEMBLY__ $(ASFLAGS_$(ARCH))		\
-				${DEFINES} ${INCLUDES}
-TF_CFLAGS		+= 	-nostdinc -ffreestanding -Wall			\
-				-Werror -Wmissing-include-dirs			\
-				-std=c99 -Os					\
-				$(TF_CFLAGS_$(ARCH))				\
-				${DEFINES} ${INCLUDES}
-TF_CFLAGS		+=	-ffunction-sections -fdata-sections
+# common to CPP, CC, AS
+COMMON_FLAGS		=	${DEFINES} ${INCLUDES} -nostdinc		\
```

Comment:
> To stick to this convention, could you please replace '=' by ':=' in the definition of COMMON_FLAGS? Or is there a good reason why '=' is needed here?

The reason is ":=" does not work here.

We *must* use "=" because DEFINE and INCLUDES are appended later, for example, by platform.mk.   So, COMMON_FLAGS must be lazily expanded.